packages/components/src/__actions__/Button/v3/_docs/Button.stories.tsx

```diff
@@ -19,80 +21,55 @@ export default meta
 type Story = StoryObj<typeof meta>
 
 export const Playground: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
+  render: ({ children, ...otherProps }) => (
+    <Button {...otherProps}>{children}</Button>
   ),
 }
 
 export const ButtonWithIcon: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
-  ),
   args: {
-    children: (
-      <>
-        Label
-        <AddIcon role="presentation" />
-      </>
-    ),
+    children: <>Label</>,
+    icon: <Icon isPresentational name="delete" />,
   },
 }
 
-export const ButtonWithIconStart: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
-  ),
+export const ButtonWithIconEnd: Story = {
   args: {
-    children: (
-      <>
-        <TrashIcon role="presentation" />
-        Label
-      </>
-    ),
+    children: <>Label</>,
+    icon: <Icon isPresentational name="arrow_forward" shouldMirrorInRTL />,
+    iconPosition: "end",
   },
 }
 
 export const IconButton: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
-  ),
   args: {
     children: (
-      <>
-        <TrashIcon role="img" aria-label="Remove" />
-        <VisuallyHidden> Highlights: May 8, 2024</VisuallyHidden>
-      </>
+      <VisuallyHidden>Remove highlights from: May 8, 2024</VisuallyHidden>
     ),
+    icon: <Icon isPresentational name="delete" />,
   },
 }
 
-export const OnReversed: Story = {
-  render: ({ children, ...otherArgs }) => (
-    <Button {...otherArgs}>{children}</Button>
-  ),
-  args: {
-    children: (
-      <>
-        Label
-        <ChevronUpIcon role="presentation" />
-      </>
-    ),
-  },
+export const ReversedButton: Story = {
   parameters: {
     reverseColors: true,
     docs: {
       source: {
-        code: `
-          <ReversedColors isReversed={true}>
-            <Button {...otherArgs}>
-              <>
-                Label
-                <ChevronUpIcon role="presentation" />
-              </>
-            </Button>
-          </ReversedColors>
+        code: `<ReversedColors isReversed={true}>
```

Comment:
Either in the docs or here, but include the import for where ReversedColors comes from

Reply:
I call it out below this story in the docs, but I could include the import here?


To enable the reversed theme, you will need to the component or application in the ReversedColors provider, ie:

```tsx
import { Button } from "@kaizen/components/v3/actions"
import { ReversedColors } from "~components/__utilities__/v3"
// application code

return (
  <ReversedColors isReversed={true}>
    <Button {...buttonProps} />
  </ReversedColors>
)
```